pkg/utils/sha256.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"sort"
+	"strings"
+)
+
+func Sha256(valueMap map[string]string) string {
+	var keys []string
+	for k := range valueMap {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	var elements []string
+	for _, k := range keys {
+		element := fmt.Sprintf("%s=%s", k, valueMap[k])
+		elements = append(elements, element)
+	}
+	src := strings.Join(elements, "&")
+
+	m := sha256.New()
+	m.Write([]byte(src))
+	res := hex.EncodeToString(m.Sum(nil))
+	return res
```

Comment:
```suggestion
	return res[0, 128]
```
length 256 is t0o long, cut at 128 pos